packages/exa.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Exa < Package
+  description 'Modern replacement for \'ls\''
+  homepage 'https://the.exa.website'
+  version '0.9.0'
+  source_url 'https://github.com/ogham/exa/archive/v0.9.0.tar.gz'
+  source_sha256 '96e743ffac0512a278de9ca3277183536ee8b691a46ff200ec27e28108fef783'
+
+  depends_on 'cmake'
+  depends_on 'rust'
+  depends_on 'libgit2'
+
+  def self.build
+    case ARCH
+    when 'aarch64'
+      system 'rustup toolchain install 1.25.0-aarch64-unknown-linux-gnu'
+      system 'rustup default 1.25.0-aarch64-unknown-linux-gnu'
+    when 'armv7l'
+      system 'rustup toolchain install 1.25.0-armv7-unknown-linux-gnueabihf'
+      system 'rustup default 1.25.0-armv7-unknown-linux-gnueabihf'
```

Comment:
I imagine you don't need to build specifically for aarch64.  The armv7l build should also work on aarch64.